OneCommunityGlobal/HighestGoodNetworkApp
# Yao Wang - Remove the "(function under development)"

# Description
(PRIORITY HIGH) Jae: Blue Square Self-scheduler: Remove “(function under development)” and make the FAQ button work. At Profile Page>Below Blue Squares. 

## Related PRS (if any):
Related PRs [PR1542](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1542)+614, [PR1081](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1081) 

## Main changes explained:
1. Remove “(function under development)” text at top of modal
2. Make the FAQ button open a modal that is editable by an Owner
3. created a new modal "components/UserProfile/ScheduleReasonModal/FAQModal.jsx" based on the EditableInfoModal.jsx
4. Add one of those editable “i” icons above at left of the “Schedule Blue Square Reason” button

## How to test:
1. check into current branch
2. do npm install and npm run start:local to run this PR locally3. log as any user and owner user. 
5. go to dashboard→ Profile page→ button "Schedule Blue Square Reason"
6. verify text "(function under development)" is removed.
7. verify function “info circle” (Above the button "Schedule Blue Square Reason") is working, and only Owner account can edit it. 
8. verify function “FAQ Button” is working, the FAQ Button is also only editable by owner account. 

## Screenshots or videos of changes:
### Before: 
<img width="264" alt="Screenshot 2024-01-13 at 1 11 46 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/137429365/5ab4bf5c-a291-4bd9-91bb-821d45efbcf7">

<img width="467" alt="Screenshot 2024-01-13 at 1 11 55 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/137429365/ebfa9692-fb8e-4c5e-a405-22a271c4dd92">

### New function: 
any account:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/137429365/189e0db9-3119-4d44-8ca2-4450e263a676

owner account:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/137429365/455e2442-e8fd-4a0b-9e02-4044d15426fe

## Note:
Include the information the reviewers need to know.
